Packs/MongoDB/Integrations/MongoDB/MongoDB.py

```diff
@@ -433,6 +441,37 @@ def drop_collection_command(
     return f'MongoDB: Collection \'{collection}` has been successfully dropped.', None
 
 
+def pipeline_query_command(client: Client, collection: str, pipeline: str, limit: str = '50', offset: str = '0',
+                           **kwargs) -> Tuple[str, dict, list]:
+    limit = int(limit)
+    offset = int(offset)
+    try:
+        json_pipeline = validate_json_objects(json.loads(pipeline))
+        raw_response = client.pipeline_query(
+            collection=collection,
+            pipeline=json_pipeline,
+        )
+    except JSONDecodeError:
+        raise DemistoException('The `pipeline` argument is not a valid json.')
+
+    if raw_response:
+        raw_response = raw_response if len(raw_response) <= limit else raw_response[offset:(offset + limit)]
+        readable_outputs = tableToMarkdown(
+            f'Total of {len(raw_response)} entries were found in MongoDB collection `{collection}` '
+            f'with pipeline: {pipeline}:',
+            t=[entry.get('_id') for entry in raw_response],
+            headers=['_id'],
+        )
+        outputs_objects = list()
+        for item in raw_response:
+            item.update({'collection': collection})
+            outputs_objects.append(item)
+        outputs = {CONTEXT_KEY: outputs_objects}
```

Comment:
Why do we need `outputs_objects`?
```suggestion
        outputs = {CONTEXT_KEY: [item.update({'collection': collection}) for item in raw_response]}
```

Reply:
![image](https://user-images.githubusercontent.com/72339940/108841088-e4b29e00-75df-11eb-9356-5c34a4e68c64.png)
it is not working :(

Reply:
We fixed this right?

Reply:
yes